cliquet/resource/__init__.py

```diff
@@ -611,6 +611,11 @@ def _add_cache_header(self, response):
         collection_expires = self.request.registry.settings.get(setting_key)
         if collection_expires is not None:
             response.cache_expires(seconds=int(collection_expires))
+        else:
+            # Since Expires has resolution in seconds, do not use cache_expires
+            # in order to omit it.
+            response.cache_control.no_cache = True
+            response.pragma = 'no-cache'
```

Comment:
Ah, oh. To be honest, I just had a look at the underlying http library of the Pyramid framework, and copied this :) https://github.com/Pylons/webob/blob/master/webob/response.py#L953

I will remove it, and update the tests, thanks!